setup.py

```diff
@@ -55,6 +55,8 @@ def get_identifier(json):
         "ruamel.yaml>=0.15",
         "toml",
         "semver",
+        "requests",
+        "requests_mock",
```

Comment:
If we take on `requests` as a new dependency then `requests_mock` should go into `dev-requirements.txt` and not here. This is because a normal user of repo2docker doesn't need the mocks.

(In a super ideal world and in a new PR we could move from having `dev-requirements.txt` to using `setup.py`'s "dependencies for tests" machinery)

Reply:
oh yes sure, thanks